Neuro/Recording/DataStructures/SabotageTypes.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Neuro.Recording.DataStructures
+{
+    public enum SabotageTypes
+    {
+        None,
+        Reactor,
+        Comms,
+        Lights,
+        Oxygen,
+        Seismic
+    }
```

Comment:
I would say to switch back to SystemTypes. This additional enum is unnecessary and the underlaying number value of the enum element doesn't actually matter.

Reply:
What should I use as a default value for this? I could make it nullable as well but not sure what would be preferred.

Reply:
You can just use `default` I guess